src/bindings/python/tests/test_runtime/test_op.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
```

Comment:
please, move these tests to https://github.com/openvinotoolkit/openvino/blob/master/src/bindings/python/tests/test_utils/test_utils.py